tests/OSPSuite.Infrastructure.Tests/Import/DataSourceSpecs.cs

```diff
@@ -186,7 +186,7 @@ protected override void Context()
                      Name = "Time",
                      Unit = new UnitDescription("s")
                   },
-                  ColumnInfo = _columnInfos[0]
+                  ColumnInfo = _columnInfos.ElementAt(0)
```

Comment:
I think those indexes here should use keys instead. Also what is the displaName of the column info defined on line 175. It it name ? You are using it as key and it's not defined